src/generic/create-or-rerun-course/CreateOrRerunCourseForm.test.jsx

```diff
@@ -0,0 +1,146 @@
+import { getConfig, initializeMockApp } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { AppProvider } from '@edx/frontend-platform/react';
+import {
+  act,
+  fireEvent, render,
+  waitFor,
+} from '@testing-library/react';
+import MockAdapter from 'axios-mock-adapter';
+import React from 'react';
```

Comment:
Please move the React import to the first line in this file. We have all imported entities associated with react on 1 line in each application component.